### PR TITLE
credential_json - added credential data

### PR DESCRIPTION
**Pull request by kisslaay**
this is temporary JSON data for POC of login and signup

### CODE REVIEW

File: src/data/credentials.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "id": 1,
+    "email": "kt@kt.kt",
+    "password": "qqqqqq"
```
Comment (Tinku-Saini) [inline src/data/credentials.json:5]: why password is qqqqq

File: src/data/credentials.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  },
+  {
+    "id": 2,
+    "email": "kisslaay@chandra.io",
+    "password": "tinku"
```
Comment (Tinku-Saini) [inline src/data/credentials.json:10]: Why tinku ?

**Review by Tinku-Saini — CHANGES_REQUESTED** [review]

File: src/data/credentials.json (hunk 2 above)
Comment (kisslaay) [reply src/data/credentials.json]: Because I love you <3

File: src/data/credentials.json (hunk 1 above)
Comment (kisslaay) [reply src/data/credentials.json:5]: coz it's easy

**Pull request merged**